cli/command/registry.go

```diff
@@ -29,6 +29,8 @@ func ElectAuthServer(ctx context.Context, cli Cli) string {
 	serverAddress := registry.IndexServer
 	if info, err := cli.Client().Info(ctx); err != nil {
 		fmt.Fprintf(cli.Out(), "Warning: failed to get default registry endpoint from daemon (%v). Using system default: %s\n", err, serverAddress)
+	} else if info.IndexServerAddress == "" {
+		fmt.Fprintf(cli.Out(), "Warning: Empty registry endpoint from daemon. Using system default: %s\n", serverAddress)
```

Comment:
Oh, I just noticed this now. These should definitely be going to `cli.Err()` not `cli.Out()`.